homeassistant/components/transmission/sensor.py

```diff
@@ -143,28 +149,45 @@ def unit_of_measurement(self):
     @property
     def device_state_attributes(self):
         """Return the state attributes, if any."""
+        limit = self._tm_client.config_entry.options[CONF_LIMIT]
```

Comment:
This line is braking and causing error on my side:
```
Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 277, in async_update_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 317, in _async_write_ha_state
    attr.update(self.device_state_attributes or {})
  File "/usr/src/homeassistant/homeassistant/components/transmission/sensor.py", line 152, in device_state_attributes
    limit = self._tm_client.config_entry.options[CONF_LIMIT]
KeyError: 'limit'
```

Reply:
Try readding the integration

Reply:
my bad, I sitll had old yaml causing the problem..